src/views/ScaleSlider.js

```diff
@@ -0,0 +1,111 @@
+const BoneView = require("backbone-viewj");
+const _ = require("underscore");
+const $ = require("jbone");
+//const Slider = require("bootstrap-slider");
+
+const View = BoneView.extend({
+
+  
+  initialize: function(data) {
+    this.g = data.g;
+    this.listenTo(this.g.zoomer,"change:columnWidth", this.render);
+    this.toggleClass = 'msa-hide';
+    this.isVisible = true;
+    return this;
+  },
+  
+  attributes: {
+    class: "biojs_msa_scale"
+  },
+  
+  events: {
+    'change input': 'updateSlider',
+    'click button.msa-btn-close': 'hide',
+    'click button.msa-btn-open': 'show',
+    'click button[data-action]': 'clickButton',
+  },
+  
+  template: _.template('\
```

Comment:
that's a great idea!
Maybe we let the user specify his/her own template for other functionality (e.g. the labels on the left)?